### PR TITLE
Add test cases for metrics config

### LINKED ISSUE #7808: Lack of Junit test case for MetricsConfig in package org.apache.dubbo.config
- [*] I have searched the [issues](https://github.com/apache/dubbo/issues) of this repository and believe that this is not a duplicate.
- [*] I have checked the [FAQ](https://github.com/apache/dubbo/blob/master/FAQ.md) of this repository and believe that this is not a duplicate.

### Environment

* Dubbo version: 2.7.12
* Operating System version: Mac OS 10.12.1
* Java version: 1.8

### Steps to reproduce this issue

There are Junit test cases for the other subclasses of AbstractConfig like ConsumerConfig or ProviderConfig, but there are no test cases for MetricsConfig.

### PR DESCRIPTION
**Pull request by hadley0828**
## What is the purpose of the change
Fixes #7808 

## Brief changelog
add a new metricConfigTest

## Verifying this change

- [x] Make sure there is a [GitHub_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GitHub issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [x] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [x] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [x] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Review by xiaoheng1 — APPROVED** [review]
LGTM

**Review by AlbumenJ — APPROVED** [review]

File: dubbo-config/dubbo-config-api/src/test/java/org/apache/dubbo/config/MetricsConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package org.apache.dubbo.config;
```
Comment (AlbumenJ) [inline dubbo-config/dubbo-config-api/src/test/java/org/apache/dubbo/config/MetricsConfigTest.java:1]: Please add ASF license header for this file. thx

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

File: dubbo-config/dubbo-config-api/src/test/java/org/apache/dubbo/config/MetricsConfigTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dubbo.config;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
```
Comment (AlbumenJ) [inline dubbo-config/dubbo-config-api/src/test/java/org/apache/dubbo/config/MetricsConfigTest.java:23]: please do not use comma import

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

**Comment by chickenlj** [comment]
Closed no reply, reopen when it's ready.

**Pull request closed without merging**